ostelco-ios-client/ViewControllers/EKYC/NRCIVerifyViewController.swift

```diff
@@ -1,173 +0,0 @@
-//
-//  NRCIVerifyViewController.swift
```

Comment:
This just got its name changed to the correct acronym (NRIC) - apparently there were enough changes it's showing up as a delete and add rather than a diff